luda-editor/client/src/pages/sequence_edit_page/loaded/update/mod.rs

```diff
@@ -31,26 +31,21 @@ impl LoadedSequenceEditorPage {
                 }
             }
         } else if let Some(event) = event.downcast_ref::<text_input::Event>() {
-            match event {
-                text_input::Event::Focus(_)
-                | text_input::Event::Blur(_)
-                | text_input::Event::SelectionUpdated(_) => {}
```

Comment:
Is it deprecated to use all event enum variations in the match statement to prepare for addition of event enum variation?

Reply:
Not deprecated, just made sure that this code only use this event. I don't have a confident opinion about this.